### ISSUE TITLE
DB ERROR

### ISSUE THREAD

**Issue by Michuk02** [apache/superset#30239]
### Bug description

I try to import a filter based on the source of information, of which there are two, i can;t choose either of them because i get an error, provided on a screen below. I work on Apache Superset from Google Cloud of my company. 

### How to reproduce the bug

Created a dataset and cant pick any value from the fillter because i get 1011 error.

### Screenshots/recordings

![image](https://github.com/user-attachments/assets/a35d5163-e706-4cf6-a377-6742a99e100c)
![image](https://github.com/user-attachments/assets/7118c7b1-9cad-4d29-b92b-f86e726233a0)

### Superset version

master / latest-dev

### Python version

I don't know

### Node version

I don't know

### Browser

Not applicable

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [X] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by rusackas** [comment]
I agree with the bot, we need more reproduction steps. I haven't run into this, and don't quite understand how I'd test/replicate the issue. This has also been silent for > 6 months, so I'll close it as stale/not-planned, but I'm happy to revisit/reopen if you're still facing this in 4.1.2. or newer.
